crates/gitbutler-cli/src/args.rs

```diff
@@ -83,7 +83,14 @@ pub mod vbranch {
         },
         /// Add a branch to the workspace.
         Apply {
-            /// The name of the virtual branch to apply.
+            /// Whether it's a branch that we're applying.
+            ///
+            /// If a stack create from the given brach is not found a new stack is created.
+            #[clap(short = 'b', long, default_value_t = false)]
```

Comment:
There should be no need to set the default value here - it's implied to be `false`.